### DISCUSSION TITLE
Ref. Next.js 서버컴포넌트에서 pathname 에 접근하기

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#102] [아이디어]
https://velog.io/@taeyooooon/NextJS-%EC%84%9C%EB%B2%84-%EC%BB%B4%ED%8F%AC%EB%84%8C%ED%8A%B8%EC%97%90%EC%84%9C-url-pathname-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0

**Comment by Captain-Kim** [comment]
좋은 내용입니다 생소한 개념인데 어제 몇 시간만에 해결하시는 것보고 놀랐습니다 👍
